tests/test-solr.php

```diff
@@ -170,6 +170,32 @@ public function test_index_site_for_multisite() {
         $this->assertEquals(0, $indexResult);
     }
 
+    /**
+     * @group multisite
+     */
+    public function test_index_site_indexes_more_than_first_100_sites() {
+        for ($i = 0; $i < 110; $i++) {
+            $blogId = $this->factory->blog->create();
+
+            switch_to_blog($blogId);
+            $this->factory->post->create_many(1);
```

Comment:
Should we be using `create_many` here if we are only creating a single post? I believe there is an alternative factory method but I can't remember it off the top of my head sorry. Could you look into this please? Cheers

Reply:
done